src/components/select/simple-select/simple-select.test.js

```diff
@@ -0,0 +1,1022 @@
+        "have.attr",
+        "aria-expanded",
+        "true"
+      );
+      selectList().should("be.visible");
+      for (let i = 0; i < 3; i++) {
+        loader(i).should("be.visible");
+      }
+    });
+
+    it("should render a lazy loaded option when the infinte scroll prop is set", () => {
+      CypressMountWithProviders(<SimpleSelectWithInfiniteScrollComponent />);
+
+      const option = "Lazy Loaded A1";
+
+      selectText().click();
+      commonDataElementInputPreview().should(
+        "have.attr",
+        "aria-expanded",
+        "true"
+      );
+      selectList().should("be.visible");
+      for (let i = 0; i < 3; i++) {
+        loader(i).should("be.visible");
+      }
+      selectList().scrollTo("bottom").wait(250);
+      selectList().scrollTo("bottom");
+      selectList().should("be.visible");
+      for (let i = 0; i < 3; i++) {
+        loader(i).should("be.visible");
+      }
+      selectListText(option).should("be.visible");
+    });
+
+    it("should open correct list and select one when an object is already set as a value", () => {
+      CypressMountWithProviders(<SimpleSelectObjectAsValueComponent />);
+
+      const position = "first";
+      const positionValue = "Amber";
+
+      getDataElementByValue("input").should("have.attr", "value", "Green");
+      selectInput().should("have.attr", "aria-expanded", "false");
+      selectText().click();
+      selectOption(positionOfElement(position)).click();
+      getDataElementByValue("input").should(
+        "have.attr",
+        "value",
+        positionValue
+      );
+    });
+
+    it("should render list options with multiple columns", () => {
+      CypressMountWithProviders(<SimpleSelectMultipleColumnsComponent />);
+
+      const headerColumns = "3";
+      const bodyColumns = "3";
+
+      selectText().click();
+      commonDataElementInputPreview().should(
+        "have.attr",
+        "aria-expanded",
+        "true"
+      );
+      selectList().should("be.visible");
+      multiColumnsSelectListHeader()
+        .should("have.length", headerColumns)
+        .and("be.visible");
+      multiColumnsSelectListBody()
+        .should("have.length", bodyColumns)
+        .and("be.visible");
+      multiColumnsSelectListRow().should(
+        "have.css",
+        "background-color",
+        "rgb(153, 173, 183)"
+      );
+    });
+
+    it.each([
+      ["1", "favourite", "orange"],
+      ["2", "money_bag", "black"],
+      ["3", "gift", "blue"],
+    ])(
+      "should render list option %s with custom option %s icon and custom icon color %s",
+      (index, type, color) => {
+        CypressMountWithProviders(
+          <SimpleSelectCustomOptionChildrenComponent />
+        );
+
+        selectText().click();
+        commonDataElementInputPreview().should(
+          "have.attr",
+          "aria-expanded",
+          "true"
+        );
+        selectList().should("be.visible");
+        selectListCustomChild(index)
```

Comment:
should be the index passed as `string` ?

Reply:
Do you mean just using a string variable name, e.g. `option`, rather than the term `index`?